### ISSUE TITLE
Understanding factors that lead to a successful grant application

### ISSUE THREAD

**Issue by katerinabc** [OpenDataforWeb3/DataGrantsforARB#4]
The goal of the proposed bounty is to help understand what leads to a successfully funded grants. The results will help individuals structure their teams and grant proposal. In addition, it can unearth any unconscious bias or favoritism that might exists in the granting process.

Factors that should be considered is complexity of grant proposal, expertise of grantees, social presence of grantees, and their prior relationships with members of Arbitrum community.

This work could follow the process conducted by research on how scientific team's form (see Alina Lugeanu and colleagues_A computational model of team assembly in emergent scientific fields_ (2015) as an example)

This research open the doors for quantitative and qualitative researchers to collaborate by combining social network analysis, online ethnography ("netnography), and interviews. 

Output could be a prediction model and blog posts .

**Comment by Pfed-prog** [comment]
> Factors that should be considered is complexity of grant proposal, expertise of grantees, social presence of grantees, and their prior relationships with members of Arbitrum community.

Unfortunately, this is a very difficult task because contributors in the dao do not have to disclose anything because a dao is not a government organization

**Comment by epowell101** [comment]
Thank you for the issue @katerinabc !  Even the way the issue is framed is inspirational at least to me as it is more open ended and less "do this query".  

That said - [this issue] (https://github.com/OpenDataforWeb3/DataGrantsforARB/issues/8) proposes to look at grant recipients by whether they have been grant recipients elsewhere.  So that would look at least implicitly at tenure as a metric and of course at success elsewhere in gathering grant funds.  

From your issue, I can see at least a couple of directions that you might find interesting.  Please correct, modify, or ignore for that matter these ideas that come from your sentence: "Factors that should be considered is complexity of grant proposal, expertise of grantees, social presence of grantees, and their prior relationships with members of Arbitrum community."

1.  Regarding grant complexity and so forth - perhaps someone could drop all the grant proposals and grantees into their ChatGPT or similar (RAG FTW!) and look at complexity as you suggest and along the same lines - sentiment, tone, key topics and so on.  One could even look at all of the proposals across a few dimensions and see if we are tending to approve those that sound the same or if unusual proposals maybe even have a higher rate of approval (which would surprise me but maybe).  Anyway something like that?
2.  Regarding looking at the social graph of grantees - ideas might include:
- look at relationships between grantees and delegates on chain
- otherwise look at on chain groupings of behavior and again look at types that tend to get funded and whether they group together
- off chain - not sure what sources are available as Twitter / X is a bit hard.  But there must be some that someone could use.

Sorry for the length of this comment.  If I'm capturing some of what you feel might be interesting, the next question would be **whether you might want to have two or more issues and hence bounties, such as one for examining the grant applications themselves and the second would examine the relationship amongst the grantees, the delegates and so forth.** If so, feel free to split your issue into two or more such issues.  

Bottom line - THANK YOU for sharing your suggestion here.

**Comment by katerinabc** [comment]
Thanks @epowell101 for your comment. I think breaking it into two issues makes sense. I'm wondering if your second point isn't already covered in other proposed issues?

**Comment by katerinabc** [comment]
Option  1 is overlapping with: https://github.com/OpenDataforWeb3/DataGrantsforARB/issues/22

I'm unsure about the process. You want me to close this issue and re-open two new ones? As issues are connected to your dework, I don't want to clutter your dework board with unnecessary bounties
